content/guides/component-testing/slots-angular.md

```diff
@@ -0,0 +1,159 @@
+Single-slot content projection at
+[https://angular.io/guide/content-projection#single-slot](https://angular.io/guide/content-projection#single-slot)
+
+Below is a simple ButtonComponent that is using **single-slot content
+projection**.
+
+<code-block label="button.component.ts" active>
+
+```ts
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'app-button',
+    template: `
+        <button>
+            <ng-content></ng-content>
+        </button>
+    `
+})
+export class ButtonComponent {...}
+```
+
+</code-block>
+
+Now we create a **WrapperComponent** in our spec so we use to simulate passing
+in content to the `ButtonComponent` using content projection or we can mount the
+`ButtonComponent` directly using the mount template syntax.
+
+<code-group>
+<code-block label="button.component.cy.ts" active>
+
+```ts
+import { Component } from '@angular/core'
+import { ButtonComponent } from './button.component'
+
+// We create this WrapperComponent to validate
+// content projection in our tests
+@Component({
+  template: `<app-button>Click Me</app-button>`,
+})
+class WrapperComponent {}
+
+describe('ButtonComponent', () => {
+  it('can project content using a Wrapper component', () => {
+    cy.mount(WrapperComponent, {
+      declarations: [ButtonComponent],
+    })
+    cy.get('button').contains('Click Me')
+  })
+})
+```
+
+</code-block>
+<code-block label="button.component.cy.ts (with Template)">
+
+```ts
+import { ButtonComponent } from './button.component'
+
+describe('ButtonComponent', () => {
+  it('can project content using a ButtonComponent template', () => {
+    cy.mount('app-button>Click Me</app-button>', {
+      declarations: [ButtonComponent],
+    })
+    cy.get('button').contains('Click Me')
+  })
+})
+```
+
+</code-block>
+</code-group>
+
+## Multi-slot Content Projection
+
+An Angular component can also have multiple slots used for content projection.
+Each slot can specify a CSS selector that determines which content goes into
+that slot. This pattern is referred to as **multi-slot content projection**.
+With this pattern you must specify where you want the projected content to
+appear.
+
+Below is an example of CardComponent that is using **multi-slot content
+projection** and its corresponding component test. Notice we create a
+**WrapperComponent** in our spec that we use to simulate passing in content to
+the `CardComponent` using content projection.
+
+<code-group>
+<code-block label="card.component.ts" active>
+
+```ts
+import { Component, OnInit } from '@angular/core'
+
+@Component({
+  selector: 'app-card',
+  template: `
+    <div class="card">
+      <div class="card-header">
+        <ng-content select="[cardHeader]"></ng-content>
+      </div>
+
+      <div class="card-body">
+        <!--Default Slot -->
+        <ng-content></ng-content>
+      </div>
+    </div>
+  `,
+})
+export class CardComponent {}
+```
+
+</code-block>
+
+<code-block label="card.component.cy.ts">
```

Comment:
Reading through the guide, I got so used to having the `Component` and `Template` tabs that I was confused where the test for this multi-slot component was. To match the pattern above, I would have two separate code blocks.